enumfields/fields.py

```diff
@@ -70,7 +70,7 @@ def get_choices(self, include_blank=True, blank_choice=BLANK_CHOICE_DASH):
         # Force enum fields' options to use the `value` of the enumeration
         # member as the `value` of SelectFields and similar.
         return [
-            (i.value if i else i, display)
+            (i.value if isinstance(i, Enum) else i, display)
```

Comment:
An addition, if you want to do `isinstance`, I suppose it'd be better to use `ifinstance(i, self.enum)`?